index.js

```diff
@@ -2746,6 +2748,59 @@
         return filter(is(type), Just(x));
       });
 
+  //# keys :: StrMap a -> Array String
+  //.
+  //. Returns an array of the keys of the supplied StrMap.
+  //.
+  //. ```javascript
+  //. > S.keys({a: 1, b: 2, c: 3})
+  //. ['a', 'b', 'c']
```

Comment:
Let's add `.sort()` to the input, both to ensure determinism and to reinforce the comment about arbitrary ordering.